### PR TITLE
adds: backend mockups for season and standings

### PR DESCRIPTION
**Pull request by raygervais**
New Routes

/seasons/*
/standings/*

### CODE REVIEW

File: backend/routes/profile.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func ConnectProfileSubrouter(r *mux.Router) {
 	f := r.PathPrefix("/profile").Subrouter()
 
 	// "/profile/"
-	f.HandleFunc("/", AddNews).Methods("POST")
+	f.HandleFunc("/", SaveProfile).Methods("POST")
```
Comment (Grommers00) [inline backend/routes/profile.go:19]: good catch

File: mobile/lib/views/news/list.dart [hunk 2]
```diff
@@ -7,7 +7,7 @@ import 'package:mobile/services/news.dart';
 import 'details.dart';
 
 Future fetchNewsItems() async {
-  final response = await http.get('http://192.168.166.72:3000/news/');
+  final response = await http.get('http://192.168.17.15:3000/news/');
```
Comment (Grommers00) [inline mobile/lib/views/news/list.dart]: should we perhaps set this up to use the .env variable/

File: mobile/lib/views/standings/list.dart [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'package:mobile/services/season.dart';
+import 'package:mobile/services/standing.dart';
+
+Future fetchNewsItems(int seasonId) async {
+  final response =
+      await http.get('http://192.168.17.15:3000/standing/$seasonId');
```
Comment (Grommers00) [inline mobile/lib/views/standings/list.dart]: same for this

File: mobile/lib/views/news/list.dart (hunk 2 above)
Comment (raygervais) [reply mobile/lib/views/news/list.dart]: Agreed. Will update to utilize your work incorporating .env for Dart mobile app.

**Review by Grommers00 — APPROVED** [review]

**Pull request merged**